sdk/appconfiguration/app-configuration/samples/getSettingOnlyIfChanged.ts

```diff
@@ -52,7 +52,7 @@ async function cleanupSampleValues(keys: string[], client: AppConfigurationClien
 
   for await (const setting of existingSettings) {
     await client.clearReadOnly(setting);
-    await client.deleteConfigurationSetting(setting.key!, { label: setting.label });
+    await client.deleteConfigurationSetting({ key: setting.key!, label: setting.label });
```

Comment:
The fact that `setting.key!` is here makes me think that in the future we should just mark it as a required field in the interface.

Reply:
Actually, that's no longer needed. Removed. Part of this model change also makes it so we're consistently using ConfigurationSetting throughout, which doesn't allow a nullable key.